README.md

```diff
@@ -1,90 +1,186 @@
+
+### Bootstrapping the backend
+This only needs to be done once, and only when starting completely from scratch. Since the Terraform configuration includes the resources user for storing the Terraform state, we must create the resources first using local state, then add the backend configuration which makes use of those resources. Terraform is smart enough to recognize when the backend configuration chances
+
+Requirements:
+- Admin credentials for the AWS Admin GovCloud account
+
+Working locally on your GSA machine:
+- `cd terraform/backend`
+- ensure admin credentials are in environment (eg `aws-vault exec <your admin profile> bash`)
+- comment out the `backend` block in `./main.tf`
+- `terraform init`
+- `terraform plan`
+- verify the plan is correct
+- `terraform apply`
+- make note of the backend access key and secret in the outputs
+- create the backend config file as described in [Getting started](#getting-started) with the credentials from the previous step
+- uncomment the `backend` block in `./main.tf`
+- `terraform init -backend-config=../.backend-config.tfvars`
+- `terraform plan`
+- verify no changes need to be made
+- ensure `.backend-config.tfvars` will NOT be checked into version control
+
+### Bootstrapping admin accounts
+Each AWS Admin account must contain a dummy user with the ability to assume a role in a target account that will allow it manage Terraform resources. This allows us to only use one set of credentials for each platform (commercial, govcloud) and limit the required permissions.
+
+Requirements:
+- Console access to AWS Admin Commercial and GovCloud accounts
+
+In the AWS Console for each platform
+- create a user named `terraform-user`
+- add the inline policy:
+  ```
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resources": [<staging_role_arn>, <production_role_arn>, ...]
+        }
+    ]
+  }
+  ```
+  where each `<role_arn>` corresponds to the assume role arn created when bootstrapping an environment. These can be added now if they are known, or they can be added as created later on. They will look like: `arn:aws-us-gov:iam::<account id>:role/terraform-user-role`.
+
+### Bootstrapping environments
+Before using Terraform to manage resources for a Federalist environment, we need to create a role with appropriate permissions in each AWS account associated with the environment. Since a Federalist environment may need to manage resources in both Commercial and GovCloud accounts AND accounts on different AWS platforms cannot interact, we must run this step for each one. In the steps below `platform` refers to either `commercial` or `govcloud`. This only needs to be done once, when creating an environment completely from scratch. 
```

Comment:
```suggestion
Before using Terraform to manage resources for a Federalist environment, we need to create a role with appropriate permissions in each AWS account associated with the environment. Since a Federalist environment may need to manage resources in both Commercial and GovCloud accounts AND accounts on different AWS platforms cannot interact, we must run this step for each one. In the steps below, `platform` refers to either `commercial` or `govcloud`. This only needs to be done once when creating an environment completely from scratch. 
```